_projects/git_task_list.md

```diff
@@ -0,0 +1,127 @@
+finding PR that haven’t been reviewed. This projects aims to getting more
+developer doing reviews works and sharing reviews workload, by providing an easy
+way to find issues that has not been reviewed.
+
+This will also help engage the developer to keep contributing to a project where
+they can’t find any issue that they can fix or keep the newcomer issue for new
+comer as learning path and developer with more experience can contribute by
+reviewing newcomer’s work PR.
+
+The goal of this projects is a static-site javascript web page where the user
+can find Pull Request that haven’t been reviewed. User can select projects in
+Open Source hosting services like GitHub or GitLab. They also have the ability
+to select prefered programming language, tags, owner of the pull requests, and
+other metadata that is provided by the web services (GitHub/GitLab).
+
+Because the concept of this project, is the same as [GSOC prep task](http://summerofcode.github.io/gsoc-prep-tasks),
+[it](http://github.com/GSOC prep task/gsoc-prep-tasks) can be used as the
+starting point of this project.
+
+#### Known Issue Tracker Usage In GSOC Organization
+
+Issue Tracker | Usage
+--- | ---
+GitHub | 16
+Trac | 7
+Atlassian Jira | 7
+GitLab | 4
+phabricator | 4
+BugZilla | 4
+Usabilla | 2
+Redmine | 2
+
+Because of the difficulty to support all of the known issue tracker in the range
+of GSOC timeline. This project will provide support for GiHub, GitLab, And
+Phabricator.
+
+#### Identifying Task Of A Specific Organization
+
+For a hosted issue tracker (like GitHub.com & GitLab.com) it is already
+documented and we can confidently know what resource are posible and what is not
+. The process of adding support for a new organization is posible using an
+external data source such as [Wikidata](http://wikidata.org). The app can fetch
+for list of organization that use GitHub / GitLab via [Qikidata Query API](https://query.wikidata.org/)
+
+The case is different for a self hosted Organization. The accessibility is
+different. For organization that use Phabricator as issue tracker, the process
+of adding support to it will be manually for each organization to make sure that
+the API is accessible.
+
+#### Milestone
+
+##### Phase 1
+- Support adding user token
+- Use GraphQL (GitHub)
+- Add support to known GSOC 2018 organizations that are using GitHub as an issue
+  tracker.
+- Add GitLab.com support.
+  - Search by Projects.
+  - Search open issues.
+  - Search issue by tag.
+
+##### Phase 2
+- Add support to known GSOC 2018 organizations that are using GitLab.com as an issue
+  tracker.
+- Add support GitLab CE
+- Add Discourse support.
+  - Search for topics with bugs type label.
+- Add support for discourse that opened their api for public use.
+
+##### Phase 3
+- GitHub
+  - Add pull request support.
+    - Search unreviewed PR.
+    - Browse PR by authors. ([issues/20](https://github.com/summerofcode/gsoc-prep-tasks/issues/20))
```

Comment:
by phase 3, no feature should be available for only one hoster, unless the feature is *unique* to that hoster.

And PR author is a feature available on every code review hoster, so it should be a generic feature available for every hoster, so all orgs can use that feature.